### PR TITLE
Notes for weekly meeting 108

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
Weekly meeting minutes from 2017-09-14

### CODE REVIEW

File: organization/_posts/2017-09-14-startup.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+- CMS dual licensing schema with GPL v3 and Apache 2. In practice only some parts of the software could be released with Apache 2.
+This is now ‘dual licensing’, but ‘mixed licensing’ - different parts of the code released with different, compatible licenses (slides are updated to better reflect this).
+- Clarifying OpenLab licensing requirements is needed: this sets the boundary conditions for the experiments who want/need to collaborate with them. Giacomo will follow up.
+- Need to double check the compatibility of GPL/Apache with CERN's Status as an international organisation.
+
+
+## News, general matters
+
+- HSF/WLCG joint workshop: final dates decided, March 26-29. 2 sites offered to host: Naples or Orsay. Naples preferred by  GDB. Still need to check if there is a room large enough for the common day (largest mentioned at the GDB presentation is 150 which may be a little bit small). Feedback from HSF to GDB; happy with Naples if we can confirm that there is a room big enough for the joint day.
+- LHCC: happy with the CWP work but would like to ensure that the review of the drafts involves more than LHC experiments. Belle II, FAIR and LC community are 2 clear targets.
```
Comment (hegner) [inline organization/_posts/2017-09-14-startup.md]: three clear targets

File: organization/_posts/2017-09-14-startup.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+## CWP
+
+-   #### General matters
+    - Editorial Board assembled and published on the web page. Need to clarify how work concretely: several persons in the EB are more contacts to ensure that the work is reviewed by the wide community. May need to add more. Suggestions to Pete and discuss in the next meeting.
```
Comment (hegner) [inline organization/_posts/2017-09-14-startup.md]: how *to* work

File: organization/_posts/2017-09-14-startup.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+-   #### Simulation
+	- Few open points in the Simulation CWP have been resolved and now both the Executive Summary and the White Paper are available for review
+
+-   #### MachineLearning
+
+    - No news
+
+-   #### Software Development
+
+    - Received some nice review comments from Thomas Khur (Belle II), addressing these this week.
```
Comment (hegner) [inline organization/_posts/2017-09-14-startup.md]: Kuhr

**Comment by graeme-a-stewart** [comment]
Follow up with OpenLab by Giacomo is noted - did you think we needed something more formal?

File: organization/_posts/2017-09-14-startup.md (hunk 2 above)
Comment (graeme-a-stewart) [reply organization/_posts/2017-09-14-startup.md]: Done

File: organization/_posts/2017-09-14-startup.md (hunk 3 above)
Comment (graeme-a-stewart) [reply organization/_posts/2017-09-14-startup.md]: Done

**Pull request merged**